lib/rust/ensogl/component/text/src/typeface/glyph.rs

```diff
@@ -48,11 +66,32 @@ impl Glyph {
         self.color.set(color.into().into())
     }
 
+    pub fn is_bold(&self) -> bool {
+        self.style.get() & style_flag::BOLD != 0
+    }
+
+    pub fn set_bold(&self, value: bool) {
```

Comment:
We should be super careful about the naming. This is not proper bolding of the text. We should reserve the default names for the proper implementation that updates the font correctly.

Reply:
very good catch, thats really important!